apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -357,4 +390,20 @@ const styles = StyleSheet.create({
     ...iconLabel,
     opacity: 0.5,
   },
+  checklistAction: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: 20,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: "#EEEEEE",
+    marginHorizontal: 24,
+    shadowColor: "#F3F3F3",
+    shadowOffset: { width: 0, height: 4 },
+    shadowRadius: 4,
+    shadowOpacity: 1,
+    elevation: 3,
+    backgroundColor: "#fff",
```

Comment:
Could we use colors from the `color.` here instead? I know the Figma has some extra colors but I think for now we can just use the closest match -- can make the Figma and repo consistent in future as part of #910

Reply:
This one was an oversight on my part. Will fix.